server/src/test/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimUserProvisioningTests.java

```diff
@@ -1170,6 +1200,30 @@ private static void addUser(final JdbcTemplate jdbcTemplate,
         jdbcTemplate.execute(addUserSql);
     }
 
+    private static void addUser(final JdbcTemplate jdbcTemplate,
```

Comment:
Could this signature be simplified with a parameter class, like ScimUser? It's a large number of parameters.

Reply:
fixed with 0dec2a7